earth_enterprise/third_party/postgresql/README.google

```diff
@@ -1,5 +1,5 @@
 URL: http://www.postgresql.org/developer/
-Version: 8.4.22
```

Comment:
Similar changes need to be made in the README.google file for postgis.

Reply:
There are also the `README.txt`s under `src/third_party` that also have version numbers in them, and are probably outdated.  I think we can remove the `.spec` files from `src/third_party`.

Reply:
Good points. I, now, made those changes: updated readme files and removed stale and not needed .spec files.